src/atomizer.js

```diff
@@ -587,67 +395,45 @@ Atomizer.prototype.parseConfig = function (config/*:AtomizerConfig*/, options/*:
                             ')'
                         ].join('');
                     } else {
-                        parsedValue.hex = matchVal.hex;
+                        value = matchVal.hex;
                     }
                 }
                 if (matchVal.named) {
-                    if (rule.type === 'helper') {
-                        parsedValue.param = matchVal.named;
-                        namedFound = true;
-                    }
-                    // check if the named value matches any of the values registered in rules.
-                    else if (rule.rules) {
-                        // iterate rules
-                        rule.rules.some(function (keywordRule, index) {
-                            // if we find it, then add declaration
-                            if (keywordRule.suffix === matchVal.named) {
-                                // build declaration (iterate prop the value)
-                                rule.properties.forEach(function (property) {
-                                    keywordRule.values.forEach(function (value) {
-                                        /* istanbul ignore else */
-                                        if (!parsedValue.declaration) {
-                                            parsedValue.declaration = {};
-                                        }
-                                        parsedValue.declaration[property] = value;
-                                        if (match.important || (match.parent && options.namespace)) {
-                                            parsedValue.declaration[property] += ' !important';
-                                        }
-                                    });
-                                });
-                                namedFound = true;
-                                return true;
-                            }
-                        });
+                    // first check if the named value matches any of the values
+                    // registered in arguments.
+                    if (rule.arguments && index <= rule.arguments.length && Object.keys(rule.arguments[index]).indexOf(matchVal.named) >= 0) {
```

Comment:
`Object.keys` support is limited to IE9+ client side, so if we want to support IE7-8, we'll want to use lodash here.